### PR TITLE
Fix cards list navigation

### PR DESCRIPTION
**Pull request by AnkurChunekar**
- Hamburger Menu added to Navigation component
- Fixed font size for document.
- Minor styles updated for list component.

Before
nav-ham-pr-before

After
nav-ham-pr-after

### CODE REVIEW

**Review by dipteekhd — APPROVED** [review]

**Review by AtulPant2704 — APPROVED** [review]

File: components/navigation/navigation.html [hunk 1]
```diff
@@ -217,18 +220,22 @@ <h3>Standard Navigation bar</h3>
...
+                  <i class="fas fa-bars ham-icon" id="ham-icon"></i>
                   <a href="" class="brand-name">Brand</a>
                 </div>
 
-                <div class="input-wrapper">
-                    <input type="text" placeholder="search for products, brands and more..." />
+                <div class="navigation-ham-menu" id="navigation-ham-menu">
+                  <i class="fas fa-times" id="ham-close-icon"></i>
+                  <a href="">Home</a>
+                  <a href="">Products</a>
```
Comment (VishalPatil18) [inline components/navigation/navigation.html:230]: You could give value to `href`, avoid leaving attributes without value.

File: components/navigation/navigation.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+const navSideMenu = document.querySelector("#navigation-ham-menu");
 
 navMenuIcon.addEventListener("click", () => {
-navDropdownMenu.classList.toggle("active");
+    navSideMenu.classList.toggle("active");
+});
+
+navMenuCloseIcon.addEventListener("click", () => {
+    navSideMenu.classList.toggle("active");
 });
```
Comment (VishalPatil18) [inline components/navigation/navigation.js]: You could add one empty line at the end of file to avoid such warnings by Github.

**Review by VishalPatil18 — APPROVED** [review]
Just a couple of suggestions. Apart from that rest looks quite good. Great work @AnkurChunekar 👍

File: components/navigation/navigation.js (hunk 2 above)
Comment (AnkurChunekar) [reply components/navigation/navigation.js]: Thanks for your suggestions, will work on them!

**Pull request merged**